files/build.bash

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
What do you think about using `set -x`, so we see the commands being run from this script?